api/tests/acceptance/application/users/users-route_test.js

```diff
@@ -44,6 +44,10 @@ describe('Acceptance | Route | users', function () {
       expect(updatedUserAttributes.email).to.equal(`email_${user.id}@example.net`);
       expect(updatedUserAttributes.username).to.be.null;
 
+      expect(updatedUserAttributes['has-been-anonymised']).to.be.true;
```

Comment:
**nitpick**
Le nom du test est trop long, et en plus il faudrait ajouter la fonctionnalité
`it("anomymizes user, removes authentication methods and disables user's certification center and organisation memberships", async function () {`

Je propose de créer 3 it:
- anomymizes user (and log the user who asked for it)
- removes authentication methods 
- disables user's certification center and organisation memberships

Et de mettre l'appel API dans un before

Reply:
Je vais modifier

Reply:
Done